ops/ansible/roles/bfd-pipeline/templates/bfd-pipeline-service.sh.j2

```diff
@@ -1,73 +1,27 @@
-export BFD_RDA_JOB_INTERVAL_SECONDS='{{ data_pipeline_rda_job_interval_seconds }}'
-export BFD_RDA_JOB_BATCH_SIZE='{{ data_pipeline_rda_job_batch_size }}'
-export BFD_RDA_JOB_WRITE_THREAD_COUNT='{{ data_pipeline_rda_job_write_threads }}'
-{% if data_pipeline_rda_process_dlq is defined %}
-export BFD_RDA_JOB_PROCESS_DLQ='{{ data_pipeline_rda_process_dlq }}'
-{% endif %}
-{% if data_pipeline_rda_version is defined %}
-export BFD_RDA_JOB_RDA_VERSION='{{ data_pipeline_rda_version }}'
-{% endif %}
-{% if data_pipeline_rda_job_starting_fiss_seq_num is defined %}
-export BFD_RDA_JOB_STARTING_FISS_SEQ_NUM='{{ data_pipeline_rda_job_starting_fiss_seq_num }}'
-{% endif %}
-{% if data_pipeline_rda_job_starting_mcs_seq_num is defined %}
-export BFD_RDA_JOB_STARTING_MCS_SEQ_NUM='{{ data_pipeline_rda_job_starting_mcs_seq_num }}'
-{% endif %}
-export BFD_RDA_GRPC_HOST='{{ data_pipeline_rda_grpc_host }}'
-export BFD_RDA_GRPC_PORT='{{ data_pipeline_rda_grpc_port }}'
-export BFD_RDA_GRPC_MAX_IDLE_SECONDS='{{ data_pipeline_rda_grpc_max_idle_seconds }}'
-export BFD_RDA_GRPC_AUTH_TOKEN='{{ data_pipeline_rda_grpc_auth_token }}'
-export BFD_RDA_GRPC_SERVER_TYPE='{{ data_pipeline_rda_grpc_server_type }}'
-export BFD_RDA_GRPC_INPROCESS_SERVER_MODE='{{ data_pipeline_rda_grpc_inproc_server_mode }}'
-export BFD_RDA_GRPC_INPROCESS_SERVER_S3_REGION='{{ data_pipeline_rda_grpc_inproc_server_s3_region }}'
-export BFD_RDA_GRPC_INPROCESS_SERVER_S3_BUCKET='{{ data_pipeline_rda_grpc_inproc_server_s3_bucket }}'
-export BFD_RDA_GRPC_INPROCESS_SERVER_S3_DIR='{{ data_pipeline_rda_grpc_inproc_server_s3_directory }}'
-
-# Referenced by the pipeline's logback.xml file to add environment name for use in splunk.
+export BFD_CCW_JOB_ENABLED={{ enabled if pipeline_instance_type == 'ccw' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
+export BFD_RDA_JOB_ENABLED={{ enabled if pipeline_instance_type == 'rda' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
+
+export BFD_DB_URL='{{ db_url }}' {# derived from terraform control plane #}{{ '\n' }}
+{{ '#' if db_password_override|default('', true)|length<1 else '' }}export BFD_DB_PASSWORD='{{ db_password_override|default('', true) }}'                      {# application defaults to ssm db/password #}{{ '\n' }}
+{{ '#' if db_username_override|default('', true)|length<1 else '' }}export BFD_DB_USERNAME='{{ db_username_override|default('', true) }}'                      {# application defaults to ssm db/username #}{{ '\n' }}
+{{ '#' if db_auth_type_override|default('', true)|length<1 else '' }}export BFD_DB_AUTH_TYPE='{{ db_auth_type_override|default('', true) }}'                   {# application defaults to ssm db/auth_type #}{{ '\n' }}
+{{ '#' if db_max_connections_override|default('', true)|length<1 else '' }}export BFD_DB_MAX_CONNECTIONS='{{ db_max_connections_override|default('', true) }}' {# application defaults to ssm db/max_connections #}{{ '\n' }}
```

Comment:
Changes introduced in BFD-3507 have updated the corresponding parameter from `db/max_connections` to `db/hikari/max_pool_size`:

```suggestion
{{ '#' if db_max_connections_override|default('', true)|length<1 else '' }}export BFD_DB_HIKARI_MAX_POOL_SIZE='{{ db_hikari_max_pool_size_override|default('', true) }}' {# application defaults to ssm db/hikari/max_pool_size #}{{ '\n' }}
```